test/e2e/test/test.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	. "github.com/vmware-tanzu/velero/test/e2e/util/k8s"
+)
+
+type VeleroTest interface {
+	Init(client TestClient, isTestBackup bool) VeleroTest
+	CreateResources() error
+	CheckResources() error
```

Comment:
Can `CreateResources` and `CreateResources` be merged? Why do we need to check the resources again if they are already created successfully in `CreateResources`?

Reply:
ok, I'll remove it